### PR TITLE
Update personal access token docs screenshots

### PR DESCRIPTION
**Pull request by aniketh-varma**
#### Summary
Updated the screenshots for the personal access tokens docs

#### Ticket Link
Fixes https://github.com/mattermost/docs/issues/6653

### CODE REVIEW

File: site/content/integrate/reference/personal-access-token/_index.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ Personal access tokens function similar to session tokens and can be used by int
 2. Identify the account you want to create a personal access token with. You may optionally create a new user account for your integration, such as for a bot account. By default, only System Admins have permissions to create a personal access token.
 3. To create an access token with a non-admin account, you must first give it the appropriate permissions. Go to **System Console > User Management > Users**, search for the user account, then select **Manage Roles** from the dropdown.
 
-    ![image](access_token_manage_roles.png)
+    ![image](access_token_enable.png)
 
 4. Select **Allow this account to generate personal access tokens.**
 
-    ![image](access_tokens_additional_roles.png)
+    ![image](access_token_profile.png)
```
Comment (cwarnermm) [inline site/content/integrate/reference/personal-access-token/_index.md:21]: @aniketh-varma

1. This screenshot isn't correct. Please refer to the [existing docs page](https://developers.mattermost.com/integrate/reference/personal-access-token/) as a reference.
2. Avoid renaming screenshot files. Please reuse the same filename.
3. Ensure that file extensions are lowercase (not PNG).

File: site/content/integrate/reference/personal-access-token/_index.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ Personal access tokens function similar to session tokens and can be used by int
 2. Identify the account you want to create a personal access token with. You may optionally create a new user account for your integration, such as for a bot account. By default, only System Admins have permissions to create a personal access token.
 3. To create an access token with a non-admin account, you must first give it the appropriate permissions. Go to **System Console > User Management > Users**, search for the user account, then select **Manage Roles** from the dropdown.
 
-    ![image](access_token_manage_roles.png)
+    ![image](access_token_enable.png)
```
Comment (cwarnermm) [inline site/content/integrate/reference/personal-access-token/_index.md:17]: @aniketh-varma

1. This screenshot isn't correct. Please refer to the [existing docs page](https://developers.mattermost.com/integrate/reference/personal-access-token/) as a reference.
2. Avoid renaming screenshot files. Please reuse the same filename.

**Review by cwarnermm — CHANGES_REQUESTED** [review]

**Review by hanzei — CHANGES_REQUESTED** [review]
Blocking on https://github.com/mattermost/mattermost-developer-documentation/pull/1310#discussion_r1368679487

**Comment by cwarnermm** [comment]
@aniketh-varma - Are you open to applying the feedback provided in this PR?

**Comment by cwarnermm** [comment]
@aniketh-varma - Are you open to applying the feedback provided in this PR?

**Comment by cwarnermm** [comment]
@aniketh-varma - Are you open to applying the feedback provided in this PR?

**Comment by cwarnermm** [comment]
Closing due to inactivity.

**Pull request closed without merging**